src/js/components/DataFilter/DataFilter.js

```diff
@@ -121,9 +129,11 @@ export const DataFilter = ({
     }
   }
 
-  return inputOnly ? (
-    content
-  ) : (
+  if (noForm) content = <DataForm footer={false}>{content}</DataForm>;
```

Comment:
To align with DataView, we'd wrap with DataForm instead of FormField when `noForm`.